backend/src/common/config/base-config.package.ts

```diff
@@ -26,7 +29,7 @@ class BaseConfig implements Config {
         this.logger.info('.env file found and successfully parsed!');
     }
 
-    private get envSchema(): TConfig<EnvironmentSchema> {
```

Comment:
do we need to change it?

Reply:
No, i was getting an error after resolving a merge conflict but now I realize it was only local. My bad